### PR TITLE
Multiple EntityManagers prefixes

### PR DESCRIPTION
**Pull request by matesko**
Update TablePrefix.php

Update TablePrefixTrait.php

Update Vresion.php

### CODE REVIEW

**Review by bobdenotter — APPROVED** [review]
Thanks, @matesko 👍

**Comment by bobdenotter** [comment]
I wonder why the tests aren't running on this PR, though.. 🤔

File: src/Doctrine/Version.php [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Version
     public function __construct(Connection $connection, string $tablePrefix = 'bolt')
     {
         $this->connection = $connection;
+        $tablePrefix = is_array($tablePrefix) ? $tablePrefix['default'] : $tablePrefix;
```
Comment (bobdenotter) [inline src/Doctrine/Version.php:26]: `strict_types` is true, so `tablePrefix` can never be an array, right?

**Review by bobdenotter — CHANGES_REQUESTED** [review]

File: src/Doctrine/Version.php (hunk 1 above)
Comment (matesko) [reply src/Doctrine/Version.php:26]: Yes, you are corect! I updated the code

Comment (bobdenotter) [reply src/Doctrine/Version.php:26]: `string|array` won't work. Union types are only allowed in PHP 8 and up.. Perhaps do it like this, then? 

```php
    /**
     * @param string|array $tablePrefix
     */
    public function __construct(Connection $connection, $tablePrefix = 'bolt')
```

Comment (matesko) [reply src/Doctrine/Version.php:26]: Of course, pushed...

**Review by bobdenotter — APPROVED** [review]
Thanks, @matesko

**Pull request merged**